.github/workflows/pr_label_check.yml

```diff
@@ -0,0 +1,25 @@
+name: PR label check
+
+on:
+  pull_request:
+    types:
+    - opened
+    - edited
+    - labeled
+    - unlabeled
+    - synchronize
```

Comment:
Nice touch, I didn't realize the defaults were `opened, synchronize, and reopened`.